packages/flutter/lib/src/cupertino/activity_indicator.dart

```diff
@@ -128,13 +143,16 @@ class _CupertinoActivityIndicatorPainter extends CustomPainter {
     canvas.save();
     canvas.translate(size.width / 2.0, size.height / 2.0);
 
+    // Rotate so that the top tick is the first to be drawn.
+    canvas.rotate(math.pi / 2);
```

Comment:
optional: but this function would look cleaner if we just drew from the top first (adjust the tickFundamentalRRect if needed) instead of having to rotate twice in the paint.

Reply:
Fixed

Reply:
Note that I had to update a testcase to reflect this change because the testcase assumes that the 9 o'clock tick is the one that gets drawn first. I added a comment in the testcase for clarity.